conf/schemas.go

```diff
@@ -55,21 +56,23 @@ func (s *Schemas) BuildIndex() {
 	for _, schema := range s.raw {
 		for pos := range schema.Retentions.Rets {
 			s.index = append(s.index, Schema{
-				Name:          schema.Name,
-				Pattern:       schema.Pattern,
-				Retentions:    schema.Retentions.Sub(pos),
-				Priority:      schema.Priority,
-				ReorderWindow: schema.ReorderWindow,
+				Name:               schema.Name,
+				Pattern:            schema.Pattern,
+				Retentions:         schema.Retentions.Sub(pos),
+				Priority:           schema.Priority,
+				ReorderWindow:      schema.ReorderWindow,
+				ReorderAllowUpdate: schema.ReorderAllowUpdate,
 			})
 		}
 	}
 	// add the default schema
 	for pos := range s.DefaultSchema.Retentions.Rets {
 		s.index = append(s.index, Schema{
-			Name:       s.DefaultSchema.Name,
-			Pattern:    s.DefaultSchema.Pattern,
-			Retentions: s.DefaultSchema.Retentions.Sub(pos),
-			Priority:   s.DefaultSchema.Priority,
+			Name:               s.DefaultSchema.Name,
+			Pattern:            s.DefaultSchema.Pattern,
+			Retentions:         s.DefaultSchema.Retentions.Sub(pos),
+			Priority:           s.DefaultSchema.Priority,
```

Comment:
we should probably include `ReorderWindow:      schema.ReorderWindow,` here - for consistency and in case we ever update the default schema to set the ROB

Reply:
done in 8059c85